data/json/mapgen/ws_regional_dump.json

```diff
@@ -215,30 +215,30 @@
-        ",|.|.,.,.,,..,.,.,,,.,,,,.,.,...,.,...,....,.,..,..,.,...,...,...,..,.,.,.,,.,..,........,.,,|,|.",
-        ".|,-------------------------------------------------------------------------------------------.|,",
-        ",|.,..,..,.,...,,..,..,...,,.,..,...,....,...,.,...,.,.,.,..,...,...,.,.,.,.,.,..,..,...,.,.,.,|.",
-        ",-----------------------------------------------------------------------------------------------,",
-        "..,....,,..,,..,...,..,..2.,..,,,..,,,,,..,...,.,2,.,.,..,,,...,,,..,,..,2.,..,..,...,...,..,.,.,"
+        ".|,|.,.g.p^g.^gGG.g^.g.G^2G.p.g..pg..p.g......^..2...........a.aapppppppp2pa.a........,..,.,|.|,",
+        ",|.|.^.,.gp^g^gp^g.gpg^gpg.g.G.pg.pGp.g.....^..........a...a^apppppppppppppa^a^a........,.,.|,|,",
+        ".|,|,.,^..g.p^gpGgG^gg^g^pGpp.g.^g.ggp..............^..a..aapppppppppppppppppa^^......,.,.,.|.|.",
+        ",|,|.^,...g..g^.^p^g..G^p.g.pG.^...G...^....^..............^aapppppppppppppppaa.........,.,.|,|,",
+        ".|,|.,..^...p.Gg^g.p.g^.gpgp^.g.^g.g.p...............^...a.^aaappppppppappppa^a........,.,..|.|.",
+        ",|.|,.....^..g..^^.gpgp^gp^..G.g.^.G...^............^^......^appppppppppaapa.^.^.........,.,|.|.",
+        ".|,|.,,.,^...^..Gg.g..p^..g.^.g..g........................a..^pppppppppa^.a^............,.,.|,|,",
+        ",|,|.,.,.^..p..g...p.g^g..Gp.pg^.^g..p.^.....................^pppapppaa^...............,.,.,|.|.",
+        ",|.|,.,......^...g..G.^.g.^.g.^.g....................^.....a.apppaapaa^.................,.,.|,|,",
+        ",|.|.,.,.^.........^.g....gp^g^.p^gp................^^^....a.^apaaaaa^.a...............,...,|.|.",
+        ".|,|..,....^...^.......g..g.g.....p...^..............^........aa^.^a...................,.,,.|,|,",
+        ",|,|.,.,,...^......g......g...p....p......................^.....aa....a...............,..,..|.|.",
+        ".|.|.,,,......^......^.........^.........................^^^....aa..a..................,.,.,|,|.",
+        ",|.|.,..,.................^.^....p........................^.............................,.,.|.|,",
+        ".|,|..,......................p...........p...................^^.....^^.................,.,.,|,|.",
+        ",|.|..,.,.................p..................p................^....^^^^..................,.,|.|,",
+        ".|.|.,,,.................................p...........,....,..,......^^...................,..|,|.",
+        ",|.|.,,..,......,...........,.,...,....,...,..,...,.......,....,..,..,..,..,,.,.,...,..,...,|.|.",
+        ".|,|.,.,.,.,.,...,.,.,....,.,.,.,.,.,....,.,.,....,..,.,...,.,.,.,..,..,...,.,..,..,,.,..,..|,|,",
+        ",|.|.,.,.,,..,.,.,,,.,,,,.,.,...,.,...,....,.,..,..,.,...,...,...,..,.,.,.,,.,..,.......,.,,|,|.",
+        ".|,------------------------------------------------------------------------------------------.|,",
+        ",|.,..,..,.,...,,..,..,...,,.,..,...,....,...,.,...,.,.,.,..,...,...,.,.,.,.,.,..,.....,.,.,.,|.",
```

Comment:
On the other rows, you removed a `.`, not a `,`?

Reply:
As above, it was just removing one column (maybe not even the same one).